browser-test/src/admin/admin_question_list.test.ts

```diff
@@ -159,6 +160,61 @@ test.describe('Admin question list', () => {
     await validateScreenshot(page, 'questions-list-sort-dropdown-lastmodified')
   })
 
+  test('questions used in published disabled programs is still shown in phrase `used in # program`', async () => {
+    const {page, adminQuestions, adminPrograms} = ctx
+    await enableFeatureFlag(page, 'disabled_visibility_condition_enabled')
+    await loginAsAdmin(page)
+    // Set the questionText to the same as questionName to make validation easier since questionBankNames()
+    // returns the questionText.
+    await adminQuestions.addTextQuestion({
+      questionName: 'b',
+      questionText: 'b',
+    })
+    await adminQuestions.addTextQuestion({
+      questionName: 'a',
+      questionText: 'a',
+    })
+    await adminQuestions.addTextQuestion({
+      questionName: 'c',
+      questionText: 'c',
+    })
+
+    await adminPrograms.addDisabledProgram('program-disabled')
+    // Add question a and b to the disabled program
+    await adminPrograms.editProgramBlock(
+      'program-disabled',
+      'dummy description',
+      ['a', 'b'],
+    )
+    await adminPrograms.publishProgram('program-disabled')
+    // // Add question a, b, and c to the other program
+    await adminPrograms.addAndPublishProgramWithQuestions(
+      ['a', 'b', 'c'],
+      'program-two',
+    )
+
+    await adminQuestions.gotoAdminQuestionsPage()
+    await validateScreenshot(
+      page,
+      'questions-list-sort-dropdown-disabled-programs',
+    )
+    await adminQuestions.expectQuestionProgramReferencesText({
+      questionName: 'a',
+      expectedProgramReferencesText: 'Used in 2 programs.',
```

Comment:
question (blocking): Can you confirm with product (Isabel and Sijia) that we don't want to separate out the disabled program count for "Used in 1 program"? It seems confusing to me to separate out the count for "Added to" but not "Used in". The screenshots in your TDD suggested it would be separated out for "Used in", but I'm seeing now those screenshots might be confusing both cases. I went back and read through the PRD and it wasn't clear there either, so I think we need clarification from product.

Reply:
Definitely I will confirm with product about the phrase and include it if a PR for   `Used in # programs` is needed.
But I don't think it should be blocking this PR. As the description suggested that this PR is only about the phrase `Added to # disabled programs`.

Reply:
thanks! please remove the snapshot it generated also.